css/styles.css

```diff
@@ -71,10 +71,12 @@ textarea {
   padding: 0.3rem 0.6rem;
   border-radius: 4px;
   border: none;
+  cursor: pointer;
 }
 
-.button:hover {
+.fas.fa-edit {
```

Comment:
You don't ever want to add styling to classes from other libraries. If another developer on your team wanted to use this icon, they might wonder why it suddenly has a pointer cursor. Rather, apply this styling to a custom class, and add that class to your edit icon element

Reply:
Yeah makes sense! Thank you!